core/lib/zksync_core/src/sync_layer/tests.rs

```diff
@@ -156,7 +157,11 @@ impl StateKeeperHandles {
         ensure_genesis(&mut pool.access_storage().await.unwrap()).await;
 
         let sync_state = SyncState::new();
+        let (miniblock_sealer, miniblock_sealer_handle) = MiniblockSealer::new(pool.clone(), 0);
```

Comment:
Do the tests work with non-zero capacity (e.g., 1)? If they do, I'd prefer to use it, so that the tests more closely match real conditions.